Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,39 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@{{plainid}}\.'
+      scope: storage.modifier.annotation.scala
+      push:
+        - match: '{{plainid}}\.'
+          scope: storage.modifier.annotation.scala
+        - match: '{{plainid}}'
+          scope: storage.modifier.annotation.scala
+          set: annotation-parameters
+    - match: '@{{plainid}}'
+      scope: storage.modifier.annotation.scala
+      push: annotation-parameters
+
+  annotation-parameters:
+    - match: \(
+      scope: punctuation.definition.arguments.annotation.begin.scala
+      push:
+        - match: ','
+          scope: punctuation.separator.arguments.annotation.scala
+        - include: main
+        - match: \)
+          scope: punctuation.definition.arguments.annotation.end.scala
+          pop: true
+    - match: \[
+      scope: punctuation.definition.arguments.annotation.begin.scala
+      push:
+        - match: \]
+          scope: punctuation.definition.arguments.annotation.end.scala
+          pop: true
+        - include: delimited-type-expression
+    - match: (?!\(|\[)
```

Comment:
Is whitespace allowed between `@identifier` and `(`? If so, this should allow `\s+` also.